### PR TITLE
Migration to MySQL Database 

### PR DESCRIPTION
**Pull request by kamalq97**
The purpose of this PR is help with migrating from the existing SQLite the new MySQL database.
Changelog:
- Added a **ReadMe.md** file to explain the changes made and the setup processes. _Please read it and feel free to make changes to it if you'd like._
- Modified **settings.py** to run using a MySQL database server on the localhost for the development machines
- Added a special **settings_server.py** file for the production server. This can set as the default Django app settings file by defining an environmental variable on the server, as explained in the **ReadMe.md** file

### CODE REVIEW

File: myrails/configuration/Readme.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# MySQL Database Setup
```
Comment (murradkh) [inline myrails/configuration/Readme.md:1]: one issue I faced when installing MySQL on my machine, getting errors on installing the requirement `mysqlclient==2.0.1`
Screenshot 2020-11-10 013447

File: myrails/configuration/dev.cfg [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[debug]
```
Comment (murradkh) [inline myrails/configuration/dev.cfg]: if you want to add this file to gitignore as you suggested before, and to replace this file name with "`dev-example.cfg"`

File: myrails/settings.py [hunk 3]
```diff
@@ -75,16 +82,8 @@
 # https://docs.djangoproject.com/en/2.2/ref/settings/#databases
 
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
-    },
-    'OPTIONS': {
-
-        'timeout': 20,
```
Comment (murradkh) [inline myrails/settings.py:84]: in the config file, you have a `connect_timeout` variable is that's okay?

**Review by murradkh — COMMENTED** [review]
amazing! organized and informative
did you test the changes in production?

File: myrails/settings.py (hunk 3 above)
Comment (kamalq97) [reply myrails/settings.py:84]: Yes, SQLite uses the `timeout` option while MySQL uses `connect_timeout`

File: myrails/configuration/Readme.md (hunk 1 above)
Comment (kamalq97) [reply myrails/configuration/Readme.md:1]: @murradkh  Not sure what is going on, try **[these steps](https://stackoverflow.com/questions/47870628/oserror-mysql-config-not-found-when-trying-to-pip-install-mysqlclient-dja)** and let me know if you are still having the same issue

**Review by murradkh — APPROVED** [review]

**Pull request merged**